doc/Extensions/Distributed-Poller.md

```diff
@@ -108,7 +108,7 @@ Running RRDCached
 Ubuntu (/etc/default/rrdcached)
 ```conf
 OPTS="-l 0:42217"
-OPTS="$OPTS -j /var/lib/rrdcached/journal/ -F"
+OPTS="$OPTS -R -j /var/lib/rrdcached/journal/ -F"
```

Comment:
-R only works for 1.5.5 and above, can you duplicate the code block, remove -R from the old block and do:

Ubuntu (/etc/default/rrdcached) - RRDCached 1.5.5 and above.
```conf
OPTS="-l 0:42217"
OPTS="$OPTS -R -j /var/lib/rrdcached/journal/ -F"
OPTS="$OPTS -b /opt/librenms/rrd -B"
OPTS="$OPTS -w 1800 -z 900"
```